saleor/graphql/order/bulk_mutations/order_bulk_create.py

```diff
@@ -0,0 +1,1067 @@
+    order = graphene.Field(OrderType, description="Order data.")
+    errors = NonNullList(
+        OrderBulkCreateError,
+        description="List of errors occurred on create attempt.",
+    )
+
+
+class OrderBulkCreate(BaseMutation, I18nMixin):
+    count = graphene.Int(
+        required=True,
+        default_value=0,
+        description="Returns how many objects were created.",
+    )
+    results = NonNullList(
+        OrderBulkCreateResult,
+        required=True,
+        default_value=[],
+        description="List of the created orders.",
+    )
+
+    class Arguments:
+        orders = NonNullList(
+            OrderBulkCreateInput,
+            required=True,
+            description=f"Input list of orders to create. Orders limit: {MAX_ORDERS}.",
+        )
+        error_policy = ErrorPolicyEnum(
+            required=False,
+            default_value=ErrorPolicyEnum.REJECT_EVERYTHING.value,
+            description=(
+                "Policies of error handling. DEFAULT: "
+                + ErrorPolicyEnum.REJECT_EVERYTHING.name
+            ),
+        )
+
+    class Meta:
+        description = "Creates multiple orders."
+        permissions = (OrderPermissions.MANAGE_ORDERS_IMPORT,)
+        error_type_class = OrderBulkCreateError
+        error_type_field = "bulk_order_errors"
+
+    @classmethod
+    def is_datetime_valid(cls, date: datetime) -> bool:
+        return date < timezone.now() + timedelta(minutes=MINUTES_DIFF)
+
+    @classmethod
+    def validate_order_input(cls, order_input, errors: List[OrderBulkError]):
+        weight = order_input.get("weight")
+        if weight and weight.value < 0:
+            errors.append(
+                OrderBulkError(
+                    message="Product can't have negative weight.",
+                    field="weight",
+                    code=OrderBulkCreateErrorCode.INVALID,
+                )
+            )
+
+        date = order_input.get("created_at")
+        if date and not cls.is_datetime_valid(date):
+            errors.append(
+                OrderBulkError(
+                    message="Order input contains future date.",
+                    field="createdAt",
+                    code=OrderBulkCreateErrorCode.FUTURE_DATE,
+                )
+            )
+
+        if redirect_url := order_input.get("redirect_url"):
+            try:
+                validate_storefront_url(redirect_url)
+            except ValidationError as err:
+                errors.append(
+                    OrderBulkError(
+                        message=f"Invalid redirect url: {err.message}.",
+                        field="redirectUrl",
+                        code=OrderBulkCreateErrorCode.INVALID,
+                    )
+                )
+
+        # TODO validate status (wait for fulfillments)
+        # TODO validate number (?)
+        # TODO validate zones for warehouse (?)
+        # TODO validate shipping method if available (?)
```

Comment:
What action should be taken regarding this TODO?

Reply:
This will be handled in separate PRs - description updated.